src/App/App.test.tsx

```diff
@@ -40,10 +48,71 @@ describe('App', () => {
-    
-    // expect(simplifyAPIDataForSingleState).toHaveBeenCalled();
-    // expect(mockSimplifyAPIDataForSingleState).toBeCalledWith(singleState);
-  });
+  it('should render comparison details when a category is selected and the button is clicked', async () => {
+      const accessDropdown = screen.getByTestId('dropdown');
+      const sept11 = screen.getByTestId('3');
+      const viewComparisonButton = screen.getByText('View Comparison');
+      
+      userEvent.selectOptions(accessDropdown, ['sept11'] );
+      userEvent.click(viewComparisonButton);
+
+
+      const death = await waitFor(() => screen.getByText(2977));
+      expect(death).toBeInTheDocument()
+    })
+
+    it('should return home when title is clicked from comparison details page', async () => {
+      const accessDropdown = screen.getByTestId('dropdown');
+      const viewComparisonButton = screen.getByText('View Comparison');
+      const title = screen.getByText('C🦠C🦠');
+
+      userEvent.selectOptions(accessDropdown, ['sept11']);
+      userEvent.click(viewComparisonButton);
+      
+      expect(screen.queryByText('USA Overview:')).not.toBeInTheDocument();
+      expect(screen.queryByText('Cases')).not.toBeInTheDocument();
+      expect(screen.queryByText('Deaths')).not.toBeInTheDocument();
+      expect(screen.queryByText('Current Hospitalizations')).not.toBeInTheDocument();
+      
+      userEvent.click(title);
+
+      const overview = await waitFor(() => screen.getByText('USA Overview:'));
+      const cases = await waitFor(() => screen.getByText('Cases'));
+      const deaths = await waitFor(() => screen.getByText('Deaths'));
+      const currentHospitalizations = await waitFor(() => screen.getByText('Current Hospitalizations'));
+      expect(overview).toBeInTheDocument();
+      expect(cases).toBeInTheDocument();
+      expect(deaths).toBeInTheDocument();
+      expect(currentHospitalizations).toBeInTheDocument();
+    });
+
+    it('should render a new comparison on dropdown change and button click', async () => {
+      const accessDropdown = screen.getByTestId('dropdown');
+      const viewComparisonButton = screen.getByText('View Comparison');
+      
+      userEvent.selectOptions(accessDropdown, ['sept11']);
+      userEvent.click(viewComparisonButton);
+      
+      expect(screen.queryByText('USA Overview:')).not.toBeInTheDocument();
+      expect(screen.queryByText('Cases')).not.toBeInTheDocument();
+      expect(screen.queryByText('Deaths')).not.toBeInTheDocument();
+      expect(screen.queryByText('Current Hospitalizations')).not.toBeInTheDocument();
+
+      const sept11Deaths = await waitFor(() => screen.getByText(2977));
+      expect(sept11Deaths).toBeInTheDocument();
+      
+      userEvent.selectOptions(accessDropdown, ['shark-attacks']);
+      userEvent.click(viewComparisonButton);
+
+      const ddayDeaths = await waitFor(() => screen.getByText(41));
+      expect(ddayDeaths).toBeInTheDocument();
+    })
```

Comment:
We should try and get some help with this one. Also, on line 104 we need to change `shark-attacks` to `d-day`